.readme-partials.yml

```diff
@@ -0,0 +1,62 @@
+title: |-
+  The [Cloud Translation API](https://cloud.google.com/translate/docs/),
+  can dynamically translate text between thousands
+  of language pairs. The Cloud Translation API lets websites and programs
+  integrate with the translation service programmatically. The Cloud Translation
+  API is part of the larger Cloud Machine Learning API family.
+samples_body: |-
+  ### Translate V3 Beta Samples
+
+  #### Install Dependencies
+
+  From the [root directory](https://github.com/googleapis/nodejs-translate) of the client library install the dependencies:
+
+  ```
+  npm install
+  ```
+
+  Change to the samples directory, link the google-cloud/translate library from the parent, and install its dependencies:
+
+  ```
+  cd samples/
+  npm link ../
+  npm install
+  ```
+
+  #### Run the Tests
```

Comment:
Silly question - why are we telling developers how to run the sample tests in the README?  That feels like something for CONTRIBUTING at best.

Reply:
Not silly - is that where we're putting them in other repos? My argument for leaving the command in the README (perhaps in addition to CONTRIBUTING?) is that not all users are contributors, and I see it as a teaching tool to promote TDD